src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -683,22 +703,26 @@ internal IEnumerable<PackageManagerSearchElementViewModel> Search(string query)
             if (LastSync == null) return new List<PackageManagerSearchElementViewModel>();
 
             var canLogin = PackageManagerClientViewModel.AuthenticationManager.HasAuthProvider;
+            var list = new List<PackageManagerSearchElementViewModel>();
```

Comment:
This does not need to be initialized since the object allocated will be gone soon anyway. This should be good enough:

```csharp
List<PackageManagerSearchElementViewModel> list = null;
```